yaksh/views.py

```diff
@@ -1403,3 +1421,45 @@ def download_course_csv(request, course_id):
     for student in students:
         writer.writerow(student)
     return response
+
+def activate_user(request, key):
+    ci = RequestContext(request)
+    profile = get_object_or_404(Profile, activation_key=key)
```

Comment:
So if I click on the link that I received in the mail for the second time(after verifying the first time), I am again redirected to the **'enter-email-to-verify'** page. Why is it implemented that way? Since the activation key is unique to a user and the activation key constitutes a part of the url meant for activation, I should just be redirected to the **'your-email-is-already-verified'** page.